plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -171,7 +171,20 @@ SecRule REQUEST_FILENAME "@contains /apps/text/session/create" \
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
 
-
+# FP while trying to open text files in file manager
+SecRule REQUEST_FILENAME "@rx /remote\.php/dav/(?:files|comments)" \
+    "id:9508123,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    chain"
+    SecRule REQUEST_HEADERS:Content-Type "@rx ^text/plain" \
```

Comment:
I just merged this: https://github.com/coreruleset/nextcloud-rule-exclusions-plugin/pull/14

Please update this rule as follows:
 - remove chained rule
 - add this `setvar:'tx.allowed_request_content_type=%{tx.allowed_request_content_type} |text/plain|'"`

Thank you.

Reply:
By the way, i think this change should be a part of rule `9508110`.